Dockerfile.build

```diff
@@ -1,8 +1,16 @@
 FROM golang:1.18
 
+ENV GOOS="linux"
+ENV GOARCH="amd64"
+ENV CGO_ENABLED=0
+
 ARG builder_version=v0.60.0
 RUN GO111MODULE=on go install go.opentelemetry.io/collector/cmd/builder@${builder_version}
 
 ADD . /repo
 
-RUN /go/bin/builder --config=/repo/builder/builder_configuration.yaml --output-path=/repo/dist
\ No newline at end of file
+RUN /go/bin/builder --config=/repo/builder/builder_configuration.yaml --output-path=/repo/dist
+
+EXPOSE 4318
```

Comment:
I would skip this, embedded exposed ports in Dockerfiles does not work with most orchestrations anyhow